DEVELOPMENT.md

```diff
@@ -2,6 +2,15 @@
 
 This page summarizes a couple of rules to keep in mind when developing features or making changes in Bash-it.
 
+## Debugging and Logging
+
+While developing feature or making changes in general, you can log error/warning/debug
+using `_log_error` `_log_warning` and `_log_debug`. This will help you solve problems quicker
+and also propagate important notes to other users of Bash-it.
+You can see the logs by using `bash-it doctor` command to reload and see the logs.
+Alternatively, you can set `LOG_LEVEL` to `ERROR_LOG_LEVEL`, `WARNING_LOG_LEVEL` or `ALL_LOG_LEVEL`.
```

Comment:
It might be better if we call the variable something like `BASH_IT_LOG_LEVEL` - just `LOG_LEVEL` might be too generic and might possibly clash with other Bash tools that use a similarly named variable.

Reply:
Same for the log levels, maybe use something like `BASH_IT_LOG_WARN` and so on.

Reply:
done